components/todo/AddTodoCard.vue

```diff
@@ -93,6 +95,7 @@ export default {
       const errorMessage = this.checkValidation()
       if (errorMessage) {
         this.errorMessage = errorMessage
+        toast('error', this.$t('something-went-wrong'))
```

Comment:
We could add the error message here, the toast will be more specific than generic.

Reply:
Thanks Bhai, resolved.